x-pack/plugins/enterprise_search/jest.config.js

```diff
@@ -8,4 +8,11 @@ module.exports = {
   preset: '@kbn/test',
   rootDir: '../../..',
   roots: ['<rootDir>/x-pack/plugins/enterprise_search'],
+  collectCoverage: true,
+  coverageReporters: ['text'],
+  collectCoverageFrom: [
```

Comment:
Passing this through the CLI will overwrite this setting.

Reply:
Right yeah, overriding this setting via CLI is intentional for reporting specific folder coverage. Do you see that as being an issue?

Reply:
I just wanted to point it out to make sure it's not being set if it's not intentionally used. Setting it here does deviate from the defaults of the preset defined [here](https://github.com/elastic/kibana/blob/master/packages/kbn-test/jest-preset.js#L112-L119).

Reply:
Oooh, nice - I really like those presets, I wonder if there's any easy way to merge those in here 🤔 This `collectCoverageFrom` is being intentionally used here, essentially what I wanted was to to skip reporting for our `public/plugin.ts` and `server/plugin.ts` files (AFAIK there's no real meaningful way to write unit tests for those files, correct me if I'm wrong).